net-libs/libbtbb/files/libbtbb-2020.12.1-cmake4.patch

```diff
@@ -0,0 +1,13 @@
+Bump minimum version to fix build with CMake4
```

Comment:
Not sent it upstream?

Reply:
Last update was in 2022.
I'll PR but not sure if it would be noticed.

Reply:
Remember, it's not just about it getting merged. It's for the benefit of other distros and also gives a place for people to comment (possibly from other distros) to point out problems so we can be notified and fix the patch we used.